### PR TITLE
XnView Classic for Windows (+MRU)

### PR DESCRIPTION
**Pull request by Tobias-B-Besemer**
Need some help with the syntax for CleanerML cleaning a ini-file...

### CODE REVIEW

**Comment by Tobias-B-Besemer** [comment]
This is my latest version... But I don't get it why it doesn't work... Need really some help with the syntax (and a better documentation about cleaning ini-files with CleanerML ;-)).

File: pending/xnview.xml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+	CleanerML written by https://github.com/Tobias-B-Besemer
+	v0.1 done on 2018-12-19
+
+-->
+<cleaner id="xnview.new">
```
Comment (az0) [inline pending/xnview.xml]:
```suggestion
<cleaner id="xnview.new" os="windows">
```

File: pending/xnview.xml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  <label>XnView Classic for Windows</label>
+  <description>Picture Viewer</description>
+  <option id="mru">
+    <label>MRU Files and Folders</label>
+    <description>Delete list of MRU (Most recently used) Files and Folders</description>
+        <action command="ini" search="glob" path="$APPDATA\XnView\xnview.ini" name="FileName_0"/>
```
Comment (az0) [inline pending/xnview.xml]: Change all the `glob` to `file` where there is not any wildcards in the filename

```suggestion
        <action command="ini" search="file" path="$APPDATA\XnView\xnview.ini" name="FileName_0"/>
```

**Comment by az0** [comment]
You mentioned it doesn't work and you need help, so perhaps you could post the original .ini file

**Comment by Tobias-B-Besemer** [comment]
This was my first push from my master... XnView is now #151... This Pull request is +-0 and can therefor be closed...

**Pull request closed without merging**